src/Discovery/reducers.js

```diff
@@ -0,0 +1,83 @@
+import { AccessLevel, AccessSortDirection } from './Discovery';
+
+const discovery = (
+  state = {
+    selectedResources: [],
+    actionToResume: null,
+    accessFilters: {
+      [AccessLevel.ACCESSIBLE]: true,
+      [AccessLevel.UNACCESSIBLE]: true,
+      [AccessLevel.PENDING]: true,
+      [AccessLevel.NOT_AVAILABLE]: true,
+    },
+    selectedTags: {},
+    pagination: {
+      resultsPerPage: 10,
+      currentPage: 1,
+    },
+    accessSortDirection: AccessSortDirection.DESCENDING,
+  },
+  action,
+) => {
+  switch (action.type) {
+  case 'RESOURCES_SELECTED':
+    return {
+      ...state,
+      selectedResources: action.selectedResources,
+    };
+  case 'ACCESS_FILTER_SET':
+    return {
+      ...state,
+      accessFilters: {
+        ...state.accessFilters,
+        ...action.accessFilters,
+      },
+      pagination: {
+        ...state.pagination,
+        currentPage: 1,
+      },
+    };
+  case 'TAGS_SELECTED':
+    return {
+      ...state,
+      selectedTags: action.selectedTags,
+      pagination: {
+        ...state.pagination,
+        currentPage: 1,
+      },
+    };
+  case 'SEARCH_TERM_SET':
+    return {
+      ...state,
+      searchTerm: action.searchTerm,
+      pagination: {
+        ...state.pagination,
+        currentPage: 1,
+      },
+    };
+  case 'PAGINATION_SET':
+    return {
+      ...state,
+      pagination: action.pagination,
+    };
+  case 'ACCESS_SORT_DIRECTION_SET':
+    return {
+      ...state,
+      accessSortDirection: action.accessSortDirection,
+    };
+  case 'REDIRECTED_FOR_ACTION':
+    return {
+      ...state,
+      ...action.redirectState,
```

Comment:
just want to confirm that I don't see a line of code that is dispatching an action to reset this state value, which means if a user logs in, select some studies, logs out and relog in with another username, the state of table will still be presisted and that is an intended behavior?

Reply:
When I tried this out, it looks like logging out will reset the redux store completely and the filters / selections are gone. Same thing when I log back in as a different user. 
Do we persist the redux store differently in production?

Reply:
nope, I think you are right

thanks for checking 👍